core/src/mindustry/entities/abilities/MovementLightningAbility.java

```diff
@@ -0,0 +1,39 @@
+package mindustry.entities.abilities;
+
+import arc.graphics.*;
+import arc.math.*;
+import arc.util.*;
+import arc.audio.*;
+import mindustry.content.*;
+import mindustry.entities.*;
+import mindustry.gen.*;
+
+public class MovementLightningAbility extends Ability{
+    public float damage = 35f, reload = 0.15f;
+    public int length = 12;
+    public float minSpeed = 0.8f, maxSpeed = 1.2f;
+    public Effect shootEffect = Fx.sparkShoot;
+    public Sound shootSound = Sounds.spark;
+    public Color color = Color.valueOf("a9d8ff");
+    
+    MovementLightningAbility(){}
```

Comment:
Is this constructor really needed?

Reply:
Well, The blank constructor is found in the other ability `.java`s.